build.gradle

```diff
@@ -1,72 +1,521 @@
+//version: 4f3d1b46cf51f9d4ded5379b5d817a2a6f6b302d
+/*
+DO NOT CHANGE THIS FILE!
+
+Also, you may replace this file at any time if there is an update available.
+Please check https://github.com/SinTh0r4s/ExampleMod1.7.10/blob/main/build.gradle for updates.
+ */
+
+
+import com.github.jengelman.gradle.plugins.shadow.tasks.ConfigureShadowRelocation
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
```

Comment:
There is nothing to shadow. I prefer to not have this plugin here.

Reply:
The plugin is provided by the unified build script. It will always be present so we can have one build script for all mods